annoyed-alligators/socl_media/templates/terminal.html

```diff
@@ -57,19 +57,33 @@
         if(this.command == 'cls' || this.command == 'clear'){
```

Comment:
Can you make the changes of this file in `base.html` also, since that's the one being rendered on every page. I copied the same things from here.
(The indentation of `base.html` might be a bit haywire, since there was some problem in my editor when I pushed it)